pkg/roachprod/vm/azure/flags.go

```diff
@@ -33,7 +33,7 @@ type ProviderOpts struct {
 
 var defaultLocations = []string{
 	"eastus",
-	"westus",
+	"westus2",
```

Comment:
random thought/nit: should we note why we use westus2? Personally I'd be pretty confused but maybe finding the commit message is good enough?